src/config/ui-server-config.interface.ts

```diff
@@ -10,5 +10,6 @@ export class UIServerConfig extends ServerConfig {
     windowMs: number;
     max: number;
   };
-
+  // this sets the protocol to HTTPS when the complete URL is needed in the UI
+  forceHTTPSInOrigin?: boolean;
```

Comment:
We should add this also to `config/config.example.yml`... 

We also may want to consider automating this to some extent.

For instance, it seems logical to set this to `true` by default whenever you are running in Production mode.  Production mode should be using HTTPS.

Similarly, if your are running in dev mode, it seems logical to default this to `false`, but have the option to turn it on if you are running dev mode with HTTPS.

If we automate it in this manner, we should note that in the comment to say "In Production mode, this defaults to `true`.  In development mode, this defaults to `false`."